pagespeed-insights/main.js

```diff
@@ -28,8 +28,41 @@ function getConfig(request) {
 
 var fixedSchema = [
   {
-    name: 'pageSpeed',
```

Comment:
The only breaking change in this is that the old `pageSpeed` identifier is removed. If you make either the pageSpeedmobile, or pageSpeedDesktop fields have a `name` of `pageSpeed`, this won't break existing users of the connector.

Reply:
Good call - I think it defaulted to using the Desktop version - so I will keep that in. It will be a bit confusing for users though having 3 metrics called Page Speed.